sci-electronics/kicad/kicad-6.0.11.ebuild

```diff
@@ -0,0 +1,175 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_9 )
```

Comment:
Python 3.9 support will be removed from the eclasses soon :(

Unless it will be easy to backport the 3.11 compatibility fixes, I think we are best masking version 6.0.9 already in the tree for removal, and sticking with version 7.

Reply:
It has been requested by users that we keep 6 series KiCad in tree while KiCad 7 series stabilizes through a few minor revisions.

I can take a look and see if it works with Python 3.10 as a solution.

Reply:
I've checked, the scripting console does not work with Python 3.10 due to wxwidgets-4.0.7 not being compatible, as per https://bugs.gentoo.org/902205.

So let's merge as is for now. We could also stabilize 6.0.11 a bit sooner, since Python3.9 will still be supported in stable so it should work for a while. When PyPy3.9 gets dropped from stable, we can also drop KiCad 6 series.